src/core/config_opt_mod.F90

```diff
@@ -179,15 +177,15 @@ END SUBROUTINE Set_Config
    !!
    !! \ingroup core_modules
    !!!>
-   SUBROUTINE Cleanup_Config( Config, RC )
+   SUBROUTINE Cleanup_Config( RC )
       !
       ! !USES:
       !
       USE Error_Mod
       !
       ! !INPUT/OUTPUT PARAMETERS:
       !
-      TYPE(ConfigType), INTENT(INOUT) :: Config   ! Input Options object
+      ! TYPE(ConfigType), INTENT(INOUT) :: Config   ! Input Options object
```

Comment:
remove unneeded comments

Reply:
Surely this one is for sure going to be used eventually though? Here it's ready to go.